doc/user_guide/user_guide.md

```diff
@@ -90,47 +90,70 @@ The first path element is always the name of the bucket. The next elements are t
 
-BFSC supports the following environment variables for applying parameters to all subsequent commands. If an environment variable is unset then BFSC uses the corresponding default value shown in the table below. Parameters supplied on the commandline will override the environment variables.
+The password for write operations to the BucketFS is usually stored in file `/exa/etc/EXAConf`:
+```
+WritePass = <value>
+```
 
-| Parameter                       | Environment variable | Default value |
-|---------------------------------|----------------------|---------------|
-| Host address of BucketFS server | `BUCKETFS_HOST`      | `localhost`   |
-| Port                            | `BUCKETFS_PORT`      | `2580`        |
-| Write passsword                 | `BUCKETFS_PASSWORD`  | (none)        |
-| Name of root bucket             | `BUCKETFS_BUCKET`    | `default`     |
+Additionally the `<value>` is base64 encoded.  For additional convenience BFSC allows you to provide the password in base64 encoded format and let BFSC decode it with commandline flag `--decode-base64-password` or `-d`. This applies to all methods providing the password: via the profile in your configuration file, as well as via interactive prompt.
 
-Example
 ```bash
-BUCKETFS_PASSWORD=abc
-BUCKETFS_BUCKET=simba
-
-bfsc cp foo.jar bfs:drivers
+bfsc cp -d a.txt bfs:/
 ```
 
-Is identical to
-```bash
-bfsc cp foo.jar bfs://localhost:2580/simba/drivers/foo.jar
-```
+#### Configuration File
 
-#### Retriving the Password, Base64 Encoding
+Besides specifying the complete URL on the command line you can use fallbacks for some parts of the URL that apply to multiple operations.
 
-The password for write operations to the BucketFS is usually stored in file `/exa/etc/EXAConf`:
+BFSC uses the following precedence for URI parts
+1. Supplied on the command line, or entered interactively in case of the [password](#password-protected-bucket-access)
+2. Elements of your profile
+3. Hard coded default value for host and port
+
+You can define the profile in BFSC's configuration file in your home directory: `~/.bucketfs-client/config`. On Windows the home directory is `%USERPROFILE%`.
+
+The configuration file uses the INI-file syntax. It is devided into sections. Each section contains a number of lines defining a *profile*. The first line of each section specifies the name of the profile in brackets. The default profile's name is `default`. Each of the following lines of the section may assign a value to an *element*.
+
+BFCS uses the following elements of your profile with the specified hard coded default values:
+
+| Parameter                       | Profile Element | Default value |
+|---------------------------------|-----------------|---------------|
+| Host address of BucketFS server | `host`          | `localhost`   |
+| Port                            | `port`          | `2580`        |
+| Name of root bucket             | `bucket`        | (none)        |
```

Comment:
Default value could be `default`?

Reply:
Unfortunately not, see section "Name of the Root Bucket".